discord/message.py

```diff
@@ -580,6 +580,9 @@ class Message(Hashable):
     channel: Union[:class:`TextChannel`, :class:`Thread`, :class:`DMChannel`, :class:`GroupChannel`, :class:`PartialMessageable`]
         The :class:`TextChannel` or :class:`Thread` that the message was sent from.
         Could be a :class:`DMChannel` or :class:`GroupChannel` if it's a private message.
+    thread: Optional[:class:`Thread`]
+        The thread created from this message, if any.
+        .. versionadded:: 2.0
```

Comment:
I think this is the correct formatting here
```suggestion

        .. versionadded:: 2.0
```